source/type/pb_converter.cpp

```diff
@@ -529,6 +529,9 @@ void PbCreator::Filler::fill_pb_object(const nt::StopPoint* sp, pbnavitia::StopP
         fill(pm, stop_point->mutable_physical_modes());
     }
 
+    auto* farezone = stop_point->mutable_fare_zone();
```

Comment:
This should be added only if sp->fare_zone is not empty to avoid useless fields in pb (then in json)